core/sbt-cloudflow/src/main/resources/runtimes/flink/flink-console.sh

```diff
@@ -70,4 +70,4 @@ if [[ ${JAVA_VERSION} =~ ${IS_NUMBER} ]]; then
 fi
 
 echo "Starting $SERVICE as a console application on host $HOSTNAME."
-exec $JAVA_RUN $JVM_ARGS "${log_setting[@]}" ${FLINK_ENV_JAVA_OPTS} -classpath "`manglePathList "$FLINK_TM_CLASSPATH:$INTERNAL_HADOOP_CLASSPATHS"`" ${CLASS_TO_RUN} "${ARGS[@]}"
```

Comment:
@franciscolopezsancho

Reply:
Well spotted @RayRoestenburg , this was in that order to allow the user to set logging options through conf. If `${FLINK_ENV_JAVA_OPTS}` goes after  `${log_setting[@]}` no override of logging would be possible.